backend/dataall/modules/worksheets/aws/bedrock_prompts.py

```diff
@@ -0,0 +1,118 @@
+    Table name: listings 
+                Column Metadata: [{'Name': 'price', 'Type': 'bigint'}, {'Name': 'area', 'Type': 'bigint'}, {'Name': 'bedrooms', 'Type': 'bigint'}, {'Name': 'bathrooms', 'Type': 'bigint'}, {'Name': 'stories', 'Type': 'bigint'}, {'Name': 'mainroad', 'Type': 'string'}, {'Name': 'guestroom', 'Type': 'string'}, {'Name': 'basement', 'Type': 'string'}, {'Name': 'hotwaterheating', 'Type': 'string'}, {'Name': 'airconditioning', 'Type': 'string'}, {'Name': 'parking', 'Type': 'bigint'}, {'Name': 'prefarea', 'Type': 'string'}, {'Name': 'furnishingstatus', 'Type': 'string'}, {'Name': 'passengerid', 'Type': 'bigint'}, {'Name': 'survived', 'Type': 'bigint'}, {'Name': 'pclass', 'Type': 'bigint'}, {'Name': 'name', 'Type': 'string'}, {'Name': 'sex', 'Type': 'string'}, {'Name': 'age', 'Type': 'double'}, {'Name': 'sibsp', 'Type': 'bigint'}, {'Name': 'parch', 'Type': 'bigint'}, {'Name': 'ticket', 'Type': 'string'}, {'Name': 'fare', 'Type': 'double'}, {'Name': 'cabin', 'Type': 'string'}, {'Name': 'embarked', 'Type': 'string'}]
+                Partition Metadata: []
+    <context/>""",
+        'AI': """SELECT AVG(price) AS average_price
+FROM (
+  SELECT price
+  FROM dataall_homes_11p3uu8f.listings
+  WHERE bedrooms > 3
+  ORDER BY price DESC
+  LIMIT 3)""",
+    },
+    {
+        'User': """I want to get the  3 least expensive listings with more than 3 bedrooms\n\nBased on on the following glue metadata: \n            
+    <context>
+    Database name : dataall_homes_11p3uu8f
+    Table name: listings 
+                Column Metadata: [{'Name': 'price', 'Type': 'bigint'}, {'Name': 'area', 'Type': 'bigint'}, {'Name': 'bedrooms', 'Type': 'bigint'}, {'Name': 'bathrooms', 'Type': 'bigint'}, {'Name': 'stories', 'Type': 'bigint'}, {'Name': 'mainroad', 'Type': 'string'}, {'Name': 'guestroom', 'Type': 'string'}, {'Name': 'basement', 'Type': 'string'}, {'Name': 'hotwaterheating', 'Type': 'string'}, {'Name': 'airconditioning', 'Type': 'string'}, {'Name': 'parking', 'Type': 'bigint'}, {'Name': 'prefarea', 'Type': 'string'}, {'Name': 'furnishingstatus', 'Type': 'string'}, {'Name': 'passengerid', 'Type': 'bigint'}, {'Name': 'survived', 'Type': 'bigint'}, {'Name': 'pclass', 'Type': 'bigint'}, {'Name': 'name', 'Type': 'string'}, {'Name': 'sex', 'Type': 'string'}, {'Name': 'age', 'Type': 'double'}, {'Name': 'sibsp', 'Type': 'bigint'}, {'Name': 'parch', 'Type': 'bigint'}, {'Name': 'ticket', 'Type': 'string'}, {'Name': 'fare', 'Type': 'double'}, {'Name': 'cabin', 'Type': 'string'}, {'Name': 'embarked', 'Type': 'string'}]
+                Partition Metadata: []
+        <context/>""",
+        'AI': """SELECT *
+FROM dataall_homes_11p3uu8f.listings
+WHERE bedrooms > 3
+ORDER BY price ASC
+LIMIT 3;""",
+    },
+    {
+        'User': """I want to see if any letter has been sent from 900 Somerville Avenue to 2 Finnigan Street what is the content n\nBased on on the following glue metadata: \n            
+    <context>
```

Comment:
nit: are the typos on purpose? e.g. users that type fast? 
`I want to see if any letter has been sent from 900 Somerville Avenue to 2 Finnigan Street AND what is the content n\nBased ON ON the following glue metadata:`

Reply:
nope - an oversight ... fixing now